session/src/test/java/org/apache/iotdb/session/IoTDBSessionIT.java

```diff
@@ -1203,7 +1289,7 @@ private void queryForBatch() throws ClassNotFoundException, SQLException {
       }
       Assert.assertEquals(standard, resultStr.toString());
       // d1 and d2 will align
-      Assert.assertEquals(7000, count);
+      Assert.assertEquals(14000, count);
```

Comment:
It's not intuitive where 14000 comes from.

I feel that the IoTDBSessionIT is out of control gradually... 
It's time to remove duplicated methods such as insertTabletTest1-2-3.  
Besides, there are some commented lines in test(), it should be cleaned.

I create a jira issue:  https://issues.apache.org/jira/browse/IOTDB-856. 
Better to fix it in this PR.

Reply:
ok,i will fix it in that PR